src/Voxa-Dashbot.ts

```diff
@@ -41,7 +43,10 @@ export interface IVoxaDashbotConfig {
   botframework?: string;
   debug?: boolean;
   dialogflow?: string;
+  facebook?: string;
+  redact?: boolean;
   suppressSending?: boolean;
```

Comment:
one thing i noticed with our latest project, is that we need a way to make this work as a function, something that checks the userId and decides to send or not based on that.

With loader.io for example we used big range or user ids, we want to be able to not send analytics in that case, for that range of usrs. Maybe for other PR?

Reply:
@armonge I think that's why we have this property: `ignoreUsers`, it's an array of userIds that should be ignored and analytics should not be send if the userId is among those ones

Reply:
Yes, but in the case i'm talking about, the list was too large to be included in a configuration array, 

```
        "userId": "ABwppHEKYraiAED6VdrL3RrtqFcjyC7R1NOCaqDh9WUnSMjlMYNyG9gHyN9iDWEjV1Gl_S_K3_%{*:10000-99999}"
```

Reply:
Understood, I can look into that for another PR 👍